app/models/user.rb

```diff
@@ -77,4 +77,8 @@ def request_admin_status(organisation_id)
   def self.purge_deleted_users_where(query)
     User.deleted.delete_all(query)
   end
+
+  def self.superadmin_emails
+    superadmins.pluck(:email)
+  end
 end
```

Comment:
@tansaku, this needs a rspec to test functionalilty of the class method. I also need to understand how the "extract method" refactoring works so that now I can connect the class method to the "create_proposed_organisation_edit.rb" method.